django_freeradius/tests/tests_admin.py

```diff
@@ -1,12 +1,14 @@
+from unittest import skipIf
+
 from django.contrib.auth.models import User
 from django.test import Client, TestCase
 from django.urls import reverse
 
-from ..models import (Nas, RadiusAccounting, RadiusCheck, RadiusGroup,
-                      RadiusGroupCheck, RadiusGroupReply, RadiusGroupUsers,
-                      RadiusPostAuthentication, RadiusReply, RadiusUserGroup)
+from ..models import (Nas, RadiusAccounting, RadiusCheck, RadiusGroup, RadiusGroupCheck, RadiusGroupReply,
+                      RadiusGroupUsers, RadiusPostAuthentication, RadiusReply, RadiusUserGroup,)
 
 
+@skipIf(True, "env = DATABASE_URL='mysql://mysql@127.0.0.1/sample_test' SAMPLE_APP=1")
```

Comment:
To skip the entire test class, add the following code to ``setUp`` method below:

    @skipIf(os.environ.get('SAMPLE_APP', False), 'Running tests on SAMPLE_APP')
    class UserTest(TestCase):
        ...